### DISCUSSION TITLE
Meta擺脫FB力拚轉型！宣佈股票代號改為「META」

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#420] [科技新知]
這意味著fb母公司會朝更多元的元宇宙等方面發展，期待祖克柏繼續為公司帶來新的發展，但仍要把fb顧好畢竟資安問題還是很嚴重。

資料來源：https://www.google.com/amp/s/newtalk.tw/news/view/amp/2022-06-13/769300

**Comment by AngusGuei** [comment]
說到美股，我的道瓊在亂跌啦QQ
